src/ray/core_worker/reference_count.cc

```diff
@@ -753,7 +753,7 @@ int64_t ReferenceCounter::EvictLineage(int64_t min_bytes_to_evict) {
   int64_t lineage_bytes_evicted = 0;
   while (!reconstructable_owned_objects_.empty() &&
          lineage_bytes_evicted < min_bytes_to_evict) {
-    ObjectID object_id = std::move(reconstructable_owned_objects_.front());
+    ObjectID object_id = reconstructable_owned_objects_.front();
```

Comment:
Move?

Reply:
Oh Ok we briefly discussed it several weeks ago, it's trivially copiable as of now, but I still think it's better to move in case of any code change in the future.

Reply:
done